apps/files_trashbin/lib/Trashbin.php

```diff
@@ -208,8 +219,13 @@ private static function copyFilesToUser($sourcePath, $owner, $targetPath, $user,
 
 
 		if ($view->file_exists($target)) {
-			$query = \OC_DB::prepare("INSERT INTO `*PREFIX*files_trash` (`id`,`timestamp`,`location`,`user`) VALUES (?,?,?,?)");
-			$result = $query->execute([$targetFilename, $timestamp, $targetLocation, $user]);
+			$query = \OC::$server->getDatabaseConnection()->getQueryBuilder();
+			$query->insert('files_trash')
+				->set('id', $query->createNamedParameter($targetFilename))
```

Comment:
```suggestion
				->setValue('id', $query->createNamedParameter($targetFilename))
```

It's `setValue` or `values`.